test/flow/journey.test.js

```diff
@@ -35,7 +35,7 @@ const handlerTest = ({ test, options: extraOptions }) => {
 };
 
 
-describe('Journey', () => {
+describe('journey/journey', () => {
```

Comment:
why did you make this change?

Reply:
Journey moved in to it's own package... so I described where it was in the test suite. Now I realise that it's moved again. I should fix that